pom.xml

```diff
@@ -199,6 +213,23 @@
       <version>3.27.2.1</version>
     </dependency>
 
+      <dependency>
+        <groupId>org.jmockit</groupId>
+        <artifactId>jmockit</artifactId>
+        <version>1.40</version>
+        <scope>test</scope>
+      </dependency>
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <version>6.8</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
```

Comment:
There's already a json lib called jackson in the dependency (pulled in by openapi), just use it.

Reply:
Do you mean `JacksonJSONProvider` which extends ` JacksonJaxbJsonProvider`? I searched and there were not many documents about it...

Reply:
https://github.com/FasterXML/jackson